cmd/user-agent/src/pages/chapi/DIDConnect.vue

```diff
@@ -30,17 +30,17 @@ SPDX-License-Identifier: Apache-2.0
                     </div>
 
                     <md-card-content class="viewport">
-                        <p class="viewport"> Hello {{ this.walletUser.id }},</p>
-                            <span style="color: #0E9A00">{{ requestOrigin }}</span> would like to
+                        Below issuer would like to
```

Comment:
we probably shouldn’t start sentences with Below.

Reply:
also is this specific to issuers?

Reply:
Fixed the text, will push a new PR.

For now, yes this DIDconnect happens only with issuers, RP sends DIDconnect request as part of presentation exchange query, which is handled by different vue file.